drivers/SmartThings/zigbee-switch/src/configurations.lua

```diff
@@ -60,8 +63,29 @@ local devices = {
   }
 }
 
+
 local configurations = {}
 
+configurations.power_reconfig_wrapper = function(orig_function)
+  local new_init = function(driver, device)
+    local config_version = device:get_field(CONFIGURATION_VERSION_KEY)
+    if config_version == nil or config_version < driver.current_config_version then
+      if device:supports_capability(capabilities.powerMeter) then
+        if device:supports_server_cluster(clusters.ElectricalMeasurement.ID) then
+          -- Increase minimum reporting interval to 5 seconds
+          device:send(clusters.ElectricalMeasurement.attributes.ActivePower:configure_reporting(device, 5, 600, 5))
```

Comment:
I think that in the past we've run into race conditions when sending actual zigbee messages inside `init`.

Reply:
Thinking about this, the only issue I could see is if this is a child device (which is a real issue, and I will need to fix).  Do you have any more context on what the issues were?

Reply:
there was a really old issue where the hub EUI occasionally wasn't set yet when some of these sends were called, iirc

it's possible that it has been fixed

Reply:
I re-worked this to schedule a timer to then do the actual configuration.